qiskit_optimization/algorithms/optimization_algorithm.py

```diff
@@ -444,3 +445,94 @@ def _interpret(cls, x: np.ndarray,
                             variables=problem.variables,
                             status=cls._get_feasibility_status(problem, x),
                             **kwargs)
+
+    def _interpret_samples(self, problem: QuadraticProgram, raw_samples: List[SolutionSample],
```

Comment:
Because `_get_feasibility_status` is a static method, you should make it `@classmethod`.